docs/luigi-client-api.md

```diff
@@ -384,12 +383,13 @@ Conequently the following calls shall have the exact same effect:
 ##### Parameters
 
 -   `semanticSlug` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** concatenation of semantic object and action connected with a dash (-), i.e.: `<semanticObject>-<action>`
--   `params` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** an object representing all the parameters passed, i.e.: `{param1: '1', param2: 2, param3: 'value3'}`
+-   `params` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** an object representing all the parameters passed, i.e.: `{param1: '1', param2: 2, param3: 'value3'}`. (optional, default `{}`)
 
 ##### Examples
 
 ```javascript
 LuigiClient.linkManager().navigateToIntent('Sales-settings', {project: 'pr2', user: 'john'})
+LuigiClient.linkManager().navigateToIntent('Sales-settings')
```

Comment:
(n2h) Maybe we could ```{project: 'pr2', user: 'john'}``` save this in the comment? Like instead of: ```{param1: '1', param2: 2, param3: 'value3'}``` we could say: ```{param1: '1', param2: 2, param3: 'value3'} i.e. {project: 'pr2', user: 'john'}```. If that does not contradict any standards of documenting.

Reply:
Yep, agree. Changed it up, to keep it consistent.